### PR TITLE
feat: email login (WEBAPP-4445)

### PR DESCRIPTION
**Pull request by AndyLnd**

### CODE REVIEW

File: app/script/auth/page/Login.js [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+                    <CheckboxLabel>{_(loginStrings.publicComputer)}</CheckboxLabel>
+                  </Checkbox>
+                </Form>
+              </div>
+              <Columns>
+                <Column>
+                  <Link href={ROUTE.PHONE_LOGIN + window.location.search}>{_(loginStrings.forgotPassword)}</Link>
+                </Column>
+                <Column>
+                  <Link href={ROUTE.PASSWORD_RESET + window.location.search}>{_(loginStrings.phoneLogin)}</Link>
```
Comment (gregor) [inline app/script/auth/page/Login.js]: These two look swapped. Password reset also is handled on account.wire.com. This URL won't work.

File: app/script/auth/component/ClientItem.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+/*
+ * Wire
+ * Copyright (C) 2017 Wire Swiss GmbH
```
Comment (lipis) [inline app/script/auth/component/ClientItem.jsx]: 2018 :) updated the rest in #2467

File: app/script/auth/component/ClientItem.jsx [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+                    onClick={this.handleSubmit}
+                    data-uie-name="do-remove-device"
+                  />
+                </InputSubmitCombo>
+              </Form>
+            </ContainerXS>
+          )}
+        </ContainerXS>
+        {validationError && selected ? (
+          <div style={{margin: '16px 0 0 0'}}>{parseValidationErrors(validationError)}</div>
```
Comment (lipis) [inline app/script/auth/component/ClientItem.jsx:174]: The last `0` is obsolete.. :)

File: app/script/auth/component/ClientItem.jsx [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+        ) : clientError && selected ? (
+          <ErrorMessage style={{margin: '16px 0 0 0'}} data-uie-name="error-message">
```
Comment (lipis) [inline app/script/auth/component/ClientItem.jsx:176]: Same here..

**Pull request closed without merging**